lib/sie/document.rb

```diff
@@ -73,14 +73,17 @@ def add_vouchers
 
 
     def add_voucher(opts)
-      creditor      = opts.fetch(:creditor)
-      type          = opts.fetch(:type)
-      number        = opts.fetch(:number)
-      booked_on     = opts.fetch(:booked_on)
-      description   = opts.fetch(:description).slice(0, DESCRIPTION_LENGTH_MAX)
-      voucher_lines = opts.fetch(:voucher_lines)
+      number         = opts.fetch(:number)
+      booked_on      = opts.fetch(:booked_on)
+      description    = opts.fetch(:description).slice(0, DESCRIPTION_LENGTH_MAX)
+      voucher_lines  = opts.fetch(:voucher_lines)
+      voucher_series = opts.fetch(:series) do
```

Comment:
Thanks for this! I'll leave it to another co-worker who knows this library better to review/merge the bigger picture, but here's a nitpick if you don't mind ;)

We like to follow [Weirich style](https://github.com/barsoom/devbook/tree/master/styleguide#use-weirich-style-for-blocks) meaning we'd use `{}` for this block.